src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -90,8 +94,12 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
         <h2 className="AppCard__body__name">{name}</h2>
         <div className="AppCard__body__description">{description}</div>
         <div className="AppCard__body__url">{url}</div>
-        <Button className="AppCard__body__subscribe" onClick={async e => handleSubscription(e)}>
-          Subscribe
+        <Button
+          className="AppCard__body__subscribe"
+          onClick={async e => handleSubscription(e)}
+          disabled={isSubscribed}
+        >
+          {isSubscribed ? 'Subscribed' : 'Subscribe'}
```

Comment:
Why do we show subscribed app cards in the first place?

Reply:
Wouldn't we have an unsubscribe button if we're already subscribed ?

Reply:
Unsubscribing is done at the level of the subscription, through the `AppSelector`, which I'm okay with. But in an "Explore" view, there shouldn't be things you've already explored imo

Reply:
I think it might be more convenient for the end users if they can also filter the explorer apps based on subscribed/unsubscribed (which would default to all unsubscribed apps) but that's just my opinion.
Going through the AppSelector list might be a little tedious if they have a lot of subscriptions.

Reply:
> Why do we show subscribed app cards in the first place?

This was mainly me for simple state debugging, agreed that it doesn't really make sense for end users thanks for pointing out.

I've tweaked it to filter down the Explore view to exclude any apps present in `activeSubscriptions` instead